packages/gulp-bem-xjst/test/gulp-bem-xjst.test.js

```diff
@@ -34,7 +34,7 @@ describe('gulp-bem-xjst', function () {
         it('outputs bemhtml templates compiler', function () {
             var bemhtml = _eval(vinylFile.contents.toString());
 
-            expect(bemhtml.bemhtml.apply({block: 'page'})).to.be.equal('<h1 class="page">Hello, world!</h1>');
+            expect(bemhtml.BEMHTML.apply({block: 'page'})).to.be.equal('<h1 class="page">Hello, world!</h1>');
```

Comment:
as we create an instance with default opts in `before()` default `exportName` should be ok